### PR TITLE
Minor sentence changes for "Queries" chapter

### PR DESCRIPTION
**Pull request by ram535ii**
If any of these changes make the sentence incorrect (meaning-wise) just let me know :) 

Two things I noticed, but wasn't sure how to address purely cause I'm learning Relay haha. 

- [ ] Is the subselection described actually reflected in the code example?
sub selection

- [ ] I'm just not sure what this means? 
not totally clear

### CODE REVIEW

**Comment by marktani** [comment]
Thanks for this PR! As for your remarks:

1. You are right, the two parts are not in sync. I have to recheck, I think in the code we only need `id`, but previously it selected `id`, `name` and `url`. I will verify that and update this section accordingly.

2. This senctence refers to the [note concerning `setVariable` in the official docs](https://facebook.github.io/relay/docs/api-reference-relay-container.html#setvariables).

The meaning of the sentence is, that it needs some time until `this.props` is finally populated. Only then, the query is using the new variable that was set using `setVariable`. Therefor, the `setVariable` call can be seen as asynchronously. 

Does this description sound clearer to you?

Having a look at the rest of your PR now :+1:

File: content/queries/what-is-a-query.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 When we make a request to a GraphQL server to declare our specific data requirements, we are sending a *query*.
-Queries are a central part of Relay. They are consist of fields which are either *scalar* or *non-scalar*.
+Queries are a central part of Relay. They consist of fields which are either *scalar* or *non-scalar*.
 Scalar fields are primitive data types like `String` or `Int`.
 Non-scalar fields consist of non-scalar fields or scalar fields themselves.
 
 *Models* from your GraphQL schema are exposed as non-scalar fields in the query.
-Queries in GraphQL are *hierarchical*, that means that if you include a non-scalar field in a query you have to define a subselection of fields on it.
+Queries in GraphQL are *hierarchical*, that means that if you include a non-scalar field in a query you have to define a subselection of scalar fields on it.
```
Comment (marktani) [inline content/queries/what-is-a-query.md]: I think both versions lack some meaning that I want to express here. 

The new version is not 100% accurate either, as we only have to select scalar fields on the very last level. All the non-scalar fields before can only consist of non-scalar fields as well.

**Comment by ram535ii** [comment]
@marktani sorry for not getting back earlier, got swamped with HackZurich :P Would feedback on this PR still be useful?

File: content/queries/what-is-a-query.md (hunk 1 above)
Comment (ram535ii) [inline content/queries/what-is-a-query.md]: `Queries in GraphQL are *hierarchical*, that means that if you include a non-scalar field in a query, it must always have a subselection. That subselection can be a set of scalar fields, or another non-scalar field (at which point it in turn also needs a subselection).`

Its a bit verbose, but what do you think?

Comment (marktani) [inline content/queries/what-is-a-query.md]: see 9ad3d3b466a853499beb46359467d3cadb462b96 - I made it slightly clearer and am pretty happy with it now :)

Comment (ram535ii) [inline content/queries/what-is-a-query.md]: I like it

**Pull request merged**